core/jazz_cloud-logs-streamer/index.js

```diff
@@ -83,7 +83,7 @@ function transform(payload) {
     if (payload.messageType === 'CONTROL_MESSAGE') {
         return null;
     }
-    logger.info("Raw payload..:" + JSON.stringify(payload));
+    //logger.info("Raw payload..:" + JSON.stringify(payload));
```

Comment:
Any reason we are having to change the Log lines in actual code for writing unit tests?

Reply:
my bad surya. i was debugging and forgot to remove this line